packages/dds/tree/src/test/edit-manager/editManager.spec.ts

```diff
@@ -375,6 +376,28 @@ describe("EditManager", () => {
         ]);
     });
 
+    describe("Avoids unnecessary rebases", () => {
+        it("Sequenced changes that are based on the trunk should not be rebased", () => {
+            const { manager, family } = editManagerFactory();
+            const commit1: TestCommit = {
+                sessionId: peer1,
+                seqNumber: brand(1),
+                refNumber: brand(0),
+                changeset: TestChangeRebaser.mintChangeset([], 1),
+            };
+            const commit2: TestCommit = {
+                sessionId: peer1,
+                seqNumber: brand(2),
+                refNumber: brand(0),
+                changeset: TestChangeRebaser.mintChangeset([1], 2),
+            };
+            manager.addSequencedChange(commit1);
+            family.rebaser.invert = () => assert.fail("invert was called");
```

Comment:
Is it too inconvenient to create a separate ChangeFamily for this test instead of mutating the existing one?

Reply:
Good point. Turns out it wasn't too bad.